.pre-commit-config.yaml

```diff
@@ -0,0 +1,50 @@
+# See https://pre-commit.com for more information
+ci:
+  autofix_prs: false
```

Comment:
Maybe this is a question for the copier template instead: Why not autofix PRs? I always like it when stuff which is autofixable automatically fixes itself on github.